backend/api/app.py

```diff
@@ -52,7 +55,11 @@ def checkin():
                 notifications.append({"mediaType": "SMS", "phoneNumber": phone})
 
             # Run the auto_checkin script celery task in the background
```

Comment:
you were removing all the celery, so remove all the celery

Reply:
good catch!